core/schema.py

```diff
@@ -1423,6 +1423,34 @@ def async_mutate(cls, user, **data):
         return errors
 
 
+class ChangeUserLanguageMutation(OpenIMISMutation):
+    """
+    Update an existing User Language and do not change history data
+    """
+    _mutation_module = "core"
+    _mutation_class = "ChangeUserLanguageMutation"
+
+    class Input(OpenIMISMutation.Input):
+        language_id = graphene.String()
```

Comment:
Please add required attribute for this
```suggestion
        language_id = graphene.String(required=True)
```